utils/gnomad_functions.py

```diff
@@ -133,6 +133,77 @@ def read_list_data(input_file: str) -> List[str]:
     return output
 
 
+def filter_by_frequency(t: Union[hl.MatrixTable, hl.Table], frequency: float = None, allele_count: int = None,
+                        population: str = None, subpop: str = None, downsampling: int = None,
```

Comment:
should direction default really be `equal` ?

Reply:
what would you prefer?